src/nft/components/modals/Overlay.tsx

```diff
@@ -0,0 +1,15 @@
+import { Box } from '../Box'
+import * as styles from './Overlay.css'
+
+interface OverlayProps {
+  onClick?: () => void
+}
+
+export const stopPropagation = (event: any) => {
```

Comment:
This function seems error-prone, because `event` is typed as `any` but then `nativeEvent` is used from it. Could you improve the typing to prevent inadvertent errors in the future?